src/main.js

```diff
@@ -15,34 +15,76 @@
 var Util = require('./util.js');
 var WebVRPolyfill = require('./webvr-polyfill.js');
 
+var RENAMED_BOOL_CONFIG_OPTIONS = {
```

Comment:
TBH, I'd err on simplicity and just rename the flags without back compat. It's so early for WebVR, I think it's ok. Not that many people use this polyfill yet. Even fewer use the WebVRConfig options at all. And those that do shouldn't expect updates to not have somewhat breaking changes.